common/persistence/sql/sqlplugin/postgres/admin.go

```
@@ -0,0 +1,132 @@
+}
+
+// Exec executes a sql statement
+func (pdb *db) Exec(stmt string, args ...interface{}) error {
+	_, err := pdb.db.Exec(stmt, args...)
+	return err
+}
+
+// ListTables returns a list of tables in this database
+func (pdb *db) ListTables(database string) ([]string, error) {
+	var tables []string
+	err := pdb.db.Select(&tables, fmt.Sprintf(listTablesQuery))
```

golangci-lint (1.64.8)

SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead

(staticcheck)